test/e2e/ibmcloud_test.go

```diff
@@ -308,6 +308,7 @@ func newPodWithPVCFromIBMVPCBlockDriver(namespace, podName, containerName, image
 					Name:            containerName,
 					Image:           imageName,
 					ImagePullPolicy: corev1.PullAlways,
+					Ports:           []corev1.ContainerPort{{ContainerPort: 80}},
```

Comment:
Should we add that to all test suites?

Reply:
No I think, I added ContainerPort to test cases which only executes commands inside container

Reply:
I mean not just for IBM, but make this the default everywhere.

Reply:
Hi @mkulke ,
Sorry I updated the code now
- Here I added `containerport` and `ReadinessProbe` manually for ibm specific test cases - [Ref](https://github.com/confidential-containers/cloud-api-adaptor/pull/1502/files#diff-96d31bf66f849bfcdfdd93f021bd2de1bf27e2d9cf6ceb7e79f2afceef4fc3a7R312-R322)
- But for other test cases created a common function (withContainerPort) to add `containerport` and `ReadinessProbe` in [common.go](https://github.com/confidential-containers/cloud-api-adaptor/pull/1502/files#diff-2c7fc9c9c12632c5e550abeb31614356abd17710d948a8664f9d2f2163e38b0dR22-R38)

Reply:
Ok, I guess we can add it to other clouds & libvirt in separate PRs. I was just concerned to introduce different test behavior in the individual backends.

cc @kartikjoshi21 @wainersm